pkg/services/accesscontrol/database/database.go

```diff
@@ -55,6 +55,110 @@ func (s *AccessControlStore) GetUserPermissions(ctx context.Context, query acces
 	return result, err
 }
 
+// SearchUsersPermissions returns the list of user permissions indexed by UserID
```

Comment:
something to discuss: should the DB search be limited to permissions from managed roles for OSS?

Reply:
It probably should. Let's open an issue on this and deal with it in another PR.